### DISCUSSION TITLE
Schließen im Popup ausblenden

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2403] [Q&A] [answered]
Hallo,
ich möchte gerne bei allen Popups im Frontend durch das "Schließen" ausblenden...
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/styles/popup-geraete-popup#popup-schliessen-ausblenden
Leider verschwindet dieser dann auch in den Backendeinstellungen
Wie kann ich den Befehl am Besten anpassen?

.q-card__actions {
	display: none;

**Comment by mcuiobroker** [comment]
Welche  CSS Einstellung hast du genau genutzt? In der Doku steht es mit Eingrenzung auf das Gerät.

**Reply by tschortschde** [reply]
`q-card__actions {
display: none;`

Ich wollte es nicht auf einzelne Geräte eingrenzen. Ich möchte die Funktion für alle Geräte. Wie ist das umsetzbar?

**Comment by mcuiobroker** [comment, marked as answer]
Dann nimm mal vorher: jarvis-popup-container
```
.jarvis-popup-container .q-card__actions { 
     display: none;
}
```

**Reply by tschortschde** [reply]
Perfekt
Vielen vielen DANK!
